core/src/impl/Kokkos_Atomic_Generic.hpp

```diff
@@ -195,12 +200,15 @@ KOKKOS_INLINE_FUNCTION T atomic_oper_fetch(
     oldval.i = Kokkos::atomic_compare_exchange((unsigned long long int*)dest,
                                                assume.i, newval.i);
   } while (assume.i != oldval.i);
+  KOKKOS_IMPL_INTEL_INSPECTOR_END_SUPRESS_THREADING_ERRORS
+  // KOKKOS_IMPL_INTEL_INSPECTOR_END_SUPRESS_THREADING_ERRORS_FOR_RANGE(dest,
+  // sizeof(T));
```

Comment:
Why is this commented out?

Reply:
Because it's technically the correct thing to do, but adding an individual entry to the suppression table for an address range is much more expensive (and, it appears, buggier) than suppressing all threading errors. Also, there's no clear place to clean up that entry, so for a long-running program it would run out of room in the suppression table.

Reply:
Wouldn't it make sense to just delete this line then?